Products/Zuul/routers/template.py

```diff
@@ -177,7 +177,7 @@ def getThresholdDetails(self, uid):
         """
         facade = self._getFacade()
         thresholdDetails = facade.getThresholdDetails(uid)
-        form = IFormBuilder(thresholdDetails).render(fieldsets=False)
```

Comment:
Previously no thresholds used fieldsets to organize its fields so it was disabled.